src/store/index.js

```diff
@@ -0,0 +1,81 @@
+import Vue from 'vue';
+import Vuex from 'vuex';
+import * as utils from './utils';
+
+Vue.use(Vuex);
+
+
+export default new Vuex.Store({
+  state: {
+    firstStart: true,
+    APIkey: {
+      CLIENT_ID: 'zovm6afV1sSFyuLDNutQn',
+      CLIENT_SECRET: 'Zeee2pQAi37pofB9i6oYRq1b0iepGFc9r4cAjk43',
+    },
+    currentPosition: {},
+    currentWeather: {},
+    isWeatherGot: false,
```

Comment:
this is perfect candidate for `getter`

Reply:
@probil, what is a reason of using getter instead of prop here?

Reply:
@liubich It's not valid anymore. It would be valid when
```js
currentWeather: null, // or object
```
So we can compute it based on `currentWeather` this way:
```js
getters: {
  isWeatherGot: state => !!state.currentWeather
}
```

Reply:
> It's not valid anymore

Why? We can base getter on currentWeather property, for example, this way:

```javascript
getters: {
  isWeatherGot: state => !!state.currentWeather.description
}
```

Reply:
And, thanks, I understood, what is a reason.